WordPress/gutenberg
# Only exit zoom out on double clicking the section itself

## What?


Fixes Zoom Out so that only double clicking on the section block itself will exit Zoom Out mode.

Based on https://github.com/WordPress/gutenberg/pull/65702

Closes https://github.com/WordPress/gutenberg/issues/65750.

## Why?


See https://github.com/WordPress/gutenberg/issues/65750

Previously clicking on _any_ block would exit Zoom Out unexpectedly. Now only section blocks handle the event.

## How?


- Only add the listener to section blocks.
- Check the event target is the section before handling (note that the event will bubble up to the section otherwise).

## Testing Instructions





- Enable Zoom Out
- Double click on editable content _within_ a section (e.g. a heading block or something).
- See that Zoom Out does not exit
- Double click on the section block itself.
- See that Zoom Out is disengaged.

### Testing Instructions for Keyboard


## Screenshots or screencast 

https://github.com/user-attachments/assets/655e42f5-58b5-4ac5-a674-f277e11ee48a
